superflore/generators/bitbake/run.py

```diff
@@ -81,9 +81,8 @@ def main():
 
     # clone current repo
     selected_targets = active_distros
-    # try:
-    link_existing_files(args.ros_distro)
-    """
+    try:
+        link_existing_files(args.ros_distro)
```

Comment:
> This looks to be linking args.ros_distro, but the exception removes all active distros?

The function attempts to remove all distros, and has a try catch around it that just passes if it fails to remove the link.

> Related do we really need to symlink? Why don't we simple change the working directory/pass the repository location as an argument?

I think that's a much better solution. I'm more partial to that behavior. Could you propose a format for that flag?